### PR TITLE
Fix img tag html. Closes #29

### LINKED ISSUE #29: Img tags are not closed in generated markdown
**Describe the bug:**

The generated `.md` files contain links to the source code, that include an `` tag.  
When taking those markdown files and process them by html generators (which we do in our release pipelines, towards netlify), that html is considered as 'not compliant' by some generators. 

**Expected behaviour:**

The image tag should be closed , like this : ``, instead of this ``

**Steps to reproduce the issue:**

1. Generate docs, using lazydocs.
2. Navigate to the generated `.md` file
3. Verify the `` tag is not closed

**Possible Fix:**

The file `lazydocs.generation.md` should be adapted to close the img tags.

### PR DESCRIPTION
**Pull request by SamVanhoutte**
**What kind of change does this PR introduce?**

- [X] Bugfix
- [ ] New Feature
- [ ] Feature Improvement
- [ ] Refactoring
- [ ] Documentation
- [ ] Other, please describe:

**Description:**
Updates the generated markdown files, to have the `` tags correctly closed.

**Checklist:**

- [X] I have read the [CONTRIBUTING](../CONTRIBUTING.md) document.
- [X] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: workflow/2 [hunk 1]
```diff
@@ -0,0 +1 @@
+echo "_HOST_IP=$(hostname -I | cut -d ' ' -f 1)" >> $GITHUB_ENV
```
Comment (raethlein) [inline workflow/2:1]: Can this file be removed again?

File: docs/lazydocs.generation.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 <!-- markdownlint-disable -->
 
-<a href="https://github.com/ml-tooling/lazydocs/blob/main/src/lazydocs/generation.py#L0"><img align="right" style="float:right;" src="https://img.shields.io/badge/-source-cccccc?style=flat-square"></a>
+<a href="https://github.com/ml-tooling/lazydocs/blob/main/src/lazydocs/generation.py#L0"><img align="right" style="float:right;" src="https://img.shields.io/badge/-source-cccccc?style=flat-square" ></a>
```
Comment (raethlein) [inline docs/lazydocs.generation.md:3]: for some reasons, it seems that it has not worked here 😅 does it have to run again maybe?

**Comment by raethlein** [comment]
@SamVanhoutte Thank you very much for your contribution! I have left two small comments 🙂

File: workflow/2 (hunk 1 above)
Comment (SamVanhoutte) [reply workflow/2]: Done.

File: docs/lazydocs.generation.md (hunk 2 above)
Comment (SamVanhoutte) [reply docs/lazydocs.generation.md]: Should be fine now.

**Pull request merged**